testing/ncids-css-testing/stories/components/usa-breadcrumb/usa-breadcrumb.default.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import Component from '@nciocpl/ncids-twig/components/usa-breadcrumb/usa-breadcrumb.twig';
+import { TestCase } from '../../../components/test-case';
+import css from './breadcrumb.scss';
+
+import { DefaultVeryLongContent } from './content';
+
+export default {
+	title: 'Components/Breadcrumb/Default',
+	args: {},
+};
+
+const Template = (args) => Component(args);
+
+export const Default = () => (
+	<TestCase css={css} html={Template.bind({})(DefaultVeryLongContent)} />
```

Comment:
Well there's no point in the test case for very long content if everything has very long content.